### PR TITLE
AppVerifier and Obtainium exist

### PR DESCRIPTION
**Pull request by randomwithnoname**

### CODE REVIEW

**Comment by wj25czxj47bu6q** [comment]
@randomwithnoname For this PR we would like you to please [review and accept our Contributor License Agreement](https://github.com/PrivSec-dev/contributor-license-agreement).

File: content/posts/android/F-Droid Security Issues.md [hunk 1]
```diff
@@ -166,13 +166,7 @@ So far, you have been presented with referenced facts that are easily verifiable
...
-
-This is how you may proceed to get the app certificate:
-
-```
-apksigner verify --print-certs --verbose myCoolApp.apk
-```
+F-Droid is often seen as the only way to get and support open-source apps: that is not the case. Sure, F-Droid could help you in finding FOSS apps that you wouldn't otherwise have known existed. Many developers also publish their FOSS apps on the **Play Store** or their website directly. Most of the time, releases are available on **GitHub**, which is great because you can use [Obtainium](https://github.com/ImranR98/Obtainium) to track and install app updates. If downloading APKs from regular websites, you can use [AppVerifier](https://github.com/soupslurpr/AppVerifier) to validate the authenticity by comparing the certificate fingerprint against the fingerprint from another source (it wouldn't matter otherwise). Obtainium will automatically open the share menu and encourage you to share it with AppVerifier if you've got it installed (and haven't turned off that feature in Obtainium).
```
Comment (wj25czxj47bu6q) [inline content/posts/android/F-Droid Security Issues.md:169]: While mentioning Obtainium and AppVerifier would be a great addition, I think we should mention them _in addition_ to the Atom feed and `apksigner` that were already mentioned.

**Review by wj25czxj47bu6q — CHANGES_REQUESTED** [review]

**Comment by randomwithnoname** [comment]
I accept
https://github.com/PrivSec-dev/contributor-license-agreement/issues/8

**Comment by wj25czxj47bu6q** [comment]
> I accept [PrivSec-dev/contributor-license-agreement#8](https://github.com/PrivSec-dev/contributor-license-agreement/issues/8)

Thanks. Please work on incorporating the aforementioned feedback so we can merge this.

**Comment by randomwithnoname** [comment]
Moved to https://github.com/PrivSec-dev/privsec.dev/pull/289

**Pull request closed without merging**